Replicator/Worker.hh

```diff
@@ -228,7 +228,18 @@ namespace litecore { namespace repl {
 
 #pragma mark - INSTANCE DATA:
     protected:
-        void setMsgCollection(blip::MessageBuilder& msg, CollectionIndex);
+        static CollectionIndex getCollectionIndex(const blip::MessageIn&  msgIn) {
+            long l = msgIn.intProperty(kCollectionProperty, kNotCollectionIndex);
+            Assert(l <= kNotCollectionIndex);
+            return (CollectionIndex)l;
+        }
+        void assignCollectionToMsg(blip::MessageBuilder& msg, CollectionIndex i) const {
+            if (_options->collectionAware()) {
+                msg[kCollectionProperty] = i;
+            }
+        }
+        std::pair<CollectionIndex, slice>
```

Comment:
Whacky indentation problems

Reply:
Yes, I myself was changing it numerous times, I was trying to avoid too long a line. If you have suggestion, I am all ears.

Reply:
Nevermind, my screen is smaller than usual and it was adding its own line breaks which made it look different than I thought.  No issue here.